keyboards/wolf/xox70/solder/info.json

```diff
@@ -0,0 +1,135 @@
+{
+    "manufacturer": "Thocc Supply",
+    "keyboard_name": "XOX70 Solder",
+    "maintainer": "ToastyStoemp",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "backlight": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP29", "GP28", "GP27", "GP26", "GP25", "GP2", "GP1", "GP0"],
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP3"]
+    },
+    "processor": "RP2040",
+    "backlight": {
+        "pin": "GP13"
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 2,
+        "max_brightness": 120,
+        "sleep": true
+    },
+    "ws2812": {
+        "pin": "GP12",
+        "driver": "vendor"
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0083",
+        "vid": "0x5453"
+    },
+    "layouts": {
```

Comment:
experience leads to me confirm with contributor - does this board have more than one key arrangement?

Reply:
It does:

https://cdn.shopify.com/s/files/1/0505/0328/6952/files/XOX_Layout.5.jpg:
![image](https://github.com/qmk/qmk_firmware/assets/18669334/79ed7c53-6965-4ea4-ac14-cc0a0ef61bc9)